packages/paste-core/components/sidebar/src/navigation/SidebarNavigationItem.tsx

```diff
@@ -70,13 +70,19 @@ const SidebarNavigationItem = React.forwardRef<HTMLAnchorElement, SidebarNavigat
         aria-current={selected ? 'page' : undefined}
```

Comment:
todo: pls change the description in package.json to "Sidebar Navigation controls both flat and hierarchical navigation structures within an application's sidebar."

Reply:
Navigation doesn't have it's own package.json, it's shared with Sidebar container. Do you mean to update the very top line of the page?

Reply:
ya, that